src/lib.rs

```diff
@@ -352,11 +413,11 @@ where
         self.write_many(Registers::DioMapping1, &mut val)
     }
 
-    fn reset_fifo(&mut self) -> Result<(), Ecs, Espi> {
+    pub fn reset_fifo(&mut self) -> Result<(), Ecs, Espi> {
```

Comment:
What is the reason to make this public?

Reply:
I think this is also a leftover, but I don't see an immediate reason to leave it private.